.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+## Description
```

Comment:
Ideally this is its own PR but its okay this time.

Reply:
Noted.